### PR TITLE
Simplify and deduplicate installation instructions

### PR DESCRIPTION
**Pull request by steffen**
Today I wanted to install Git LFS on a Linux-based system.
Unfortunately I had a hard time to find the complete installation instructions. 😩

## Reproduction steps

1. I first opened [git-lfs.com](https://git-lfs.com/) on my macOS machine and I did _not_ see any link to installation instructions for Linux: image
2. I then went to [GitHub's documentation for installing Git LFS](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage) which besides pointing to [git-lfs.com](https://git-lfs.com/) (where I was already unsuccessful) it also points to the [README's Getting Started](https://github.com/git-lfs/git-lfs#getting-started) section.
I was happy to find a **Linux users** mention with a link to [installation instructions on packagecloud](https://packagecloud.io/github/git-lfs/install).
3. I performed the "quick install" command: `curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash`
Unfortunately Git LFS was still not installed (I got `git: 'lfs' is not a git command. See 'git --help'.` when running `git lfs`). 😢
4. I then looked at what the "quick install" script does and noticed that it only adds the packagecloud repository to apt, but it does not install it yet.
I then ran `apt-get install git-lfs` for which I got a permission error.
5. I then ran `sudo apt-get install git-lfs` which finally installed Git LFS on my Linux system. 😅

Baffled by the unclear installation instructions I then found that there is a whole document dedicated to installing Git LFS on Linux:
https://github.com/git-lfs/git-lfs/blob/main/INSTALLING.md

This document is currently not linked to from the README (or the website at [git-lfs.com](https://git-lfs.com/)).

## Proposal ☺️

I propose to simplify and deduplicate the README's installation instructions as well as adding the missing link to [INSTALLING.md](https://github.com/git-lfs/git-lfs/blob/main/INSTALLING.md).

First I noticed that there is a **Downloading** section which starts with a "You can install ..." sentence and that there is a **Installing** section.
Both sections include information on installing from **binary** and from **source**, which seem to be redundant?

I therefore propose to consolidate the **Downloading** and **Installing** sections into one **Installing** section.
This is the main change proposal of this pull request besides adding a link to [INSTALLING.md](https://github.com/git-lfs/git-lfs/blob/main/INSTALLING.md).

I also propose to make the [INSTALLING.md](https://github.com/git-lfs/git-lfs/blob/main/INSTALLING.md) document a little bit more clear.
I removed some redundant steps information in the beginning of the document and I added "1." and "2." numbering to the sections to make it clear there are two steps.
(I also lowercased some confusing uppercase mentions of apt and yum. 😊)

_I ~may also open~ have opened a pull request for https://github.com/git-lfs/git-lfs.github.com to make these README installation instructions easier to discover: https://github.com/git-lfs/git-lfs.github.com/pull/55_

### CODE REVIEW

**Review by chrisd8088 — COMMENTED** [review]
Hey, thanks for the patch, this looks like a good improvement overall!  I had a couple of ideas and suggestions, but nothing major.  Thanks again and let me know what you think of my comments!

File: README.md
Comment (steffen) [reply README.md:60]: Hi @chrisd8088 Thank you for your review and feedback! 🙇‍♂️ ❤️ 

I found the wiki with the Installation instructions only at the end of performing my pull request changes (since it was only linked to from the "From source" section).

Now I have to ask, why are there installation instructions in the README _and_ in the wiki? 😊
From the user perspective I find it quite confusing as you don't know which is the most up-to-date and the source of truth, from the maintainer perspective I also would find it as an unnecessary overhead?

Wouldn't it make sense to consolidate these instructions in one place? 🤔
(I'm happy to help with that.)

The overall goal would be to make it as easy and straight forward for users to install Git LFS and reduce confusion and friction.

Comment (chrisd8088) [reply README.md:60]: I would assume that different folks created the two sets of instructions at different times.  I think it's probably advisable for the Wiki to refer to the canonical instructions in the repo's `README.md` rather than the other way around.

Comment (steffen) [reply README.md:60]: > I think it's probably advisable for the Wiki to refer to the canonical instructions in the repo's README.md rather than the other way around.

I agree! We can then link to the README from the wiki.

If it's 👍 for you I would work then on moving and consolidating the installation instructions in the README and update this PR accordingly.

Comment (chrisd8088) [reply README.md:60]: Well, I was only talking about the Wiki's copy of these particular notes, not the whole `INSTALLING.md` page, if that was what you were thinking.  If you don't mind restoring these notes in the `README.md`, that would get my approval -- thank you!

File: README.md
Comment (steffen) [reply README.md:64]: I made the code example inline in 6b2aa30be924e8b2835bc380186453259a39c4d6.

File: README.md
Comment (steffen) [reply README.md]: > If you don't mind restoring these notes in the README.md, that would get my approval -- thank you!

Hi @chrisd8088,

In https://github.com/git-lfs/git-lfs/pull/5260/commits/6b2aa30be924e8b2835bc380186453259a39c4d6 I have restored the "From source" section from the original `README.md`.
The only change I made is making `git lfs install` inline as suggested in https://github.com/git-lfs/git-lfs/pull/5260#discussion_r1084825389.

For convenience here is the updated rendered `README.md`:
https://github.com/steffen/git-lfs/blob/main/README.md

I think these changes should make the PR approvable.
As discussed, further potential improvements I will propose in separate bitesized pull requests over time.

Thank you again for your review and feedback! 🙇‍♂️

Comment (chrisd8088) [reply README.md]: Looks great, thank you!

**Review by chrisd8088 — APPROVED** [review]
Thanks very much for this cleanup!

**Pull request merged**
